Dropdown

nikvas06 opened the pull request:
Search bar replaced with react dropdown, does not output restaurant names from json in options yet

nikvas06 committed:
Dropdown

Search bar replaced with react dropdown, does not output restaurant names from json in options yet

nwnatale24 reviewed (approved):
Looks good to me sir. Nice work.

flashkard reviewed (approved):
Nice!

nikvas06 commented:
* added dropdown, working on returning json data to options display
* added react-select package
* created RestaurantSelect and RestaurantDropdown react components
